### ISSUE TITLE
Undesired space between beginning of align environment and preceding paragraph in LaTeX output

### ISSUE THREAD

**Issue by cemulate** [jgm/pandoc#3050]
Consider the following example in markdown:

```
This is some text.
Here's an align:
\begin{align*}
& x = y \\
& y = x
\end{align*}

More text
```

This produces the following in the PDF output

![image](https://cloud.githubusercontent.com/assets/342497/17279955/d0d8e28c-5748-11e6-9727-5d0d45ef79cb.png)

There is a large amount of whitespace between the paragraph and the following align environment.
The reason is due to the following `tex` file that is produced from `pandoc` and run through `pdflatex`:

```
\begin{document}

This is some text. Here's an align:

\begin{align*}
& x = y \\
& y = x
\end{align*}

More text

\end{document}
```

If the `tex` file is changed so that there is no extra blank line between the paragraph and the align environment, as such:

```
This is some text. Here's an align:
\begin{align*}
& x = y \\
& y = x
\end{align*}
```

Then LaTeX produces the more pleasing output PDF:

![image](https://cloud.githubusercontent.com/assets/342497/17279957/e6bbe1da-5748-11e6-8f56-65413b6d7c66.png)

Certainly, the latter output should be preferred. I played around with the source for a while today, but could only manage to remove all blank lines from the `tex` file (bad results), or leave them all in. Selectively choosing where to place blank lines may require much more familiarity with the source than I have.

**Comment by pegonma** [comment]
Try this instead:

This is some text. Here's an align:
$$
\begin{aligned}
& x = y \\
& y = x
\end{aligned}
$$

**Comment by ickc** [comment]
In general, if you don't need numbering, you should use the approach suggested by @pegonma in pandoc, since that one is treated as Math, while yours is treated as raw LaTeX. (there's also bugs in using raw LaTeX, since it is hard to parse a section of code and determine it as raw LaTeX or markdown).

I guess this issue should be closed.

**Comment by NickHu** [comment]
It seems that in pandoc 2 the `align*` environment works properly, without the additional spacing - can anyone else confirm?
